easybuild/easyblocks/generic/juliapackage.py

```diff
@@ -96,11 +96,12 @@ def install_step(self):
         """Install Julia package with Pkg"""
 
         # prepend installation directory to Julia DEPOT_PATH
+        # extensions in a bundle can share their DEPOT_PATH
         # see https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH
-        depot_path = os.getenv('JULIA_DEPOT_PATH')
-        if depot_path is not None:
-            depot_path = ':'.join([self.installdir, depot_path])
-        env.setvar('JULIA_DEPOT_PATH', depot_path)
+        depot_path = os.getenv('JULIA_DEPOT_PATH', '')
```

Comment:
We should split on `os.pathsep` here before doing the `in` check, otherwise we may get a partial match:
```python
depot_path = os.getenv('JULIA_DEPOT_PATH', [])
if depot_path:
    depot_path = depot_path.split(os.pathsep)
if self.installdir not in depot_path:
    depot_path = os.pathsep.join([x for x in ([self.installdir] + depot_path) if x])
    env.setvar('JULIA_DEPOT_PATH', depot_path)
```

Reply:
very good point, fixed in https://github.com/easybuilders/easybuild-easyblocks/pull/2869/commits/f2b4131c0821019b47d96568fc604ee0bcf20de0